src/shared/ui/Slider/components/ContentSlide/ContentSlide.tsx

```diff
@@ -0,0 +1,39 @@
+import './ContentSlide.scss';
+
+import { ContentType } from '@entities/media';
+import { makeContentPath } from '@features/content';
+import Image from 'next/image';
+import Link from 'next/link';
+
+type Props = {
+  image?: string;
+  title?: string;
+  subtitle?: string;
+  id: string;
+  contentType: ContentType;
+};
+
+function ContentSlide({ id, contentType, image = '', title = '', subtitle = '' }: Props) {
+  const contentPath = makeContentPath(id, contentType);
+
+  return (
+    <div className="content-slide">
+      {image ? (
+        <Image className="content-slide__image" src={image} alt={title} fill={true} />
```

Comment:
No need to pass `true` explicitly

https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true